docs/tasks/issuers/setup-vault.rst

```diff
@@ -207,3 +207,57 @@ For more information on ClusterIssuers, read the
 :doc:`ClusterIssuer reference docs </reference/clusterissuers>`.
 
 .. _`Subject Alternative Names`: https://en.wikipedia.org/wiki/Subject_Alternative_Name
+
+Vault Authentication with Kubernetes Service Accounts
+=====================================================
+
+This Vault authentication method uses Service Account tokens created by
+Kubernetes to authenticate requests to Vault for signing certificates. You can
+find more information on how to configure vault for Kubernetes based Service
+Account authentication in the `documentation
+<https://www.vaultproject.io/docs/auth/kubernetes.html>`__. This authentication
+expects three stanzas; a secret reference of the Service Account to use,
+an optional authentication mount path that is defaulted to `/kubernetes`, and
```

Comment:
Should be two backticks here, and is it `/kubernetes` or `kubernetes`? This contradicts the API doc slightly.